debugger/src/main/scala/org.apache.daffodil.tdml/TDML.scala

```diff
@@ -123,6 +123,8 @@ object TDML {
       .relativize(new File(path.toString()).toURI())
       .getPath()
       .toString()
+      .replaceFirst("^\\s++", "")
+      .replaceFirst("\\s++$", "")
```

Comment:
Just use `.trim()` here instead.  It will remove leading and trailing whitespace characters from strings.

Reply:
.trim() only removes a whitespace and it does not remove multiple whitespaces before and after. We also used .strip() function but it is only available post Java11 and make it noncompatible with Java8

Reply:
```
scala> "'" + "  foo\t  ".trim + "'"
val res1: String = 'foo'
```
`trim` does the right thing

Reply:
although... it is very odd to have to do this to a string that is derived from a `File` and `Path`... what's up?